governance-charter.md

```diff
@@ -132,9 +130,6 @@ criteria. Exception eligibility applications will be approved by the bootstrap
 committee by a simple majority vote. The exception process will be used as data
 points for refining the criteria for the future.
 
-Within its first year, the Governance Committee will redefine these criteria
-around a more robust idea of community membership.
```

Comment:
Suggestion: The Governance Committee may periodically review and redefine these criteria around a more robust idea of community membership.